manifests/charts/gateway/values.yaml

```diff
@@ -73,6 +73,7 @@ autoscaling:
   minReplicas: 1
   maxReplicas: 5
   targetCPUUtilizationPercentage: 80
+  autoscaleBehavior: {}
```

Comment:
Why do you need this? Are there any user cases?

Reply:
Configurable scaling behavior has been stable in k8s v1.23 [k8s document](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#configurable-scaling-behavior) . This feature has been added to other charts. I think gateway can also be added.

Reply:
This doesn't seem reasonable to me, as we are not adding features simply because they are stable or have been added to other charts. We add features when we encounter specific use cases, otherwise the charts will be full of junks.

Reply:
link: https://github.com/istio/istio/issues/47491